lib/core/keys.dart

```diff
@@ -26,8 +26,8 @@ class Keys {
    * format. Some examples of this format:
    *     `ctrl-space`, `f1`, `macctrl-a`, `shift-left`, `alt-.`
    */
-  void bind(List<String> keys, Function action) {
-    keys.forEach((key) => _bindings[key] = action);
+  void bind(List<String> keys, Function action, String description) {
```

Comment:
We might want to name this param something besides `action`, as we're creating an `Action` instance in the method. Perhaps `onInvoke`, `behavior`, `function`, ...